pkg/alertmanager/amcfg.go

```diff
@@ -294,10 +302,48 @@ func (cg *configGenerator) convertReceiver(ctx context.Context, in *monitoringv1
 		}
 	}
 
+	var webhookConfigs []*webhookConfig
+	if l := len(in.WebhookConfigs); l > 0 {
+		webhookConfigs = make([]*webhookConfig, l)
+		for i := range in.WebhookConfigs {
+			webhookConfigs[i] = cg.convertWebhookConfig(ctx, in.WebhookConfigs[i], crKey)
+		}
+	}
+
 	return &receiver{
 		Name:             prefixReceiverName(in.Name, crKey),
 		PagerdutyConfigs: pagerdutyConfigs,
+		WebhookConfigs:   webhookConfigs,
+	}
+}
+
+func (cg *configGenerator) convertWebhookConfig(ctx context.Context, in monitoringv1alpha1.WebhookConfig, crKey types.NamespacedName) *webhookConfig {
+	out := &webhookConfig{}
+
+	if in.SendResolved != nil {
+		out.VSendResolved = *in.SendResolved
 	}
+
+	if in.URLSecret != nil {
+		url, err := cg.store.getSecretKey(ctx, crKey.Namespace, *in.URLSecret)
+		if err != nil {
+			level.Warn(cg.logger).Log("msg", "failed to get url from secret for webhook, ignoring it and continuing",
```

Comment:
Hmm we're not skipping the webhook config here. And I realize that we don't do it either for PagerDuty...